lsp-pwsh.el

```diff
@@ -1,134 +1,139 @@
-    "-FeatureFlags" "@(' ')"
-    ))
-
-(defun lsp-pwsh--extra-init-params ()
-  "Return form describing parameters for language server."
-  )
-
-(defun lsp-pwsh--force-post-completion (&rest _args)
-  (advice-remove 'company-tng--supress-post-completion 'lsp-pwsh--force-post-completion)
-  nil)
-
-(defvar lsp-pwsh--major-modes '(powershell-mode))
-
-(if (fboundp 'company-lsp)
-    (advice-add 'company-tng-frontend
-                :after
-                #'(lambda (command)
-                    (when (and (eq command 'pre-command)
-                               (memq major-mode lsp-pwsh--major-modes))
-                      (advice-add 'company-tng--supress-post-completion
-                                  :after-while
-                                  'lsp-pwsh--force-post-completion)))))
-
-(lsp-register-client
- (make-lsp-client
-  :new-connection (lsp-stdio-connection 'lsp-pwsh--command)
-  :major-modes lsp-pwsh--major-modes
-  :server-id 'pwsh-ls
-  :priority 1
-  :initialization-options 'lsp-pwsh--extra-init-params
-  :notification-handlers (lsp-ht ("powerShell/executionStatusChanged" 'ignore))
-  ))
-
-(defun lsp-pwsh--filter-cr (str)
-  "Filter CR entities from STR."
-  (when (and (eq system-type 'windows-nt) str)
-      (replace-regexp-in-string "\r" "" str)))
-(advice-add 'lsp-ui-doc--extract :filter-return #'lsp-pwsh--filter-cr)
-(advice-add 'lsp-ui-sideline--extract-info :filter-return #'lsp-pwsh--filter-cr)
-
-;;; Utils
-(defconst lsp-pwsh-unzip-script "powershell -noprofile -noninteractive \
--nologo -ex bypass Expand-Archive -path '%s' -dest '%s'"
-  "Powershell script to unzip vscode extension package file.")
-
-(defcustom lsp-pwsh-github-asset-url
-  "https://github.com/%s/%s/releases/latest/download/%s"
-  "GitHub latest asset template url."
-  :group 'lsp-pwsh
-  :type 'string)
-
-(defun lsp-pwsh--get-extension (url dest)
-  "Get extension from URL and extract to DEST."
-  (let ((temp-file (make-temp-file "ext" nil ".zip")))
-    (url-copy-file url temp-file 'overwrite)
-    (if (file-exists-p dest) (delete-directory dest 'recursive))
-    (shell-command (format lsp-pwsh-unzip-script temp-file dest))))
-
-(defun lsp-pwsh-setup (&optional forced)
-  "Downloading PowerShellEditorServices to `lsp-pwsh-dir'.
-FORCED if specified."
-  (interactive "P")
-  (let ((parent-dir (file-name-directory lsp-pwsh-dir)))
-    (unless (and (not forced) (file-exists-p parent-dir))
-      (lsp-pwsh--get-extension
-       (format lsp-pwsh-github-asset-url "PowerShell" "PowerShellEditorServices" "PowerShellEditorServices.zip")
-       parent-dir)
-      (message "lsp-pwsh: Downloading done!")))
-  )
-
-(lsp-pwsh-setup)
-
-(provide 'lsp-pwsh)
-;;; lsp-pwsh.el ends here
+;;; lsp-pwsh.el --- lsp-mode client for PowerShellEditorServices  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2019  Kien Nguyen
+
+;; Author: kien.n.quang@gmail.com
+;; URL: https://github.com/kiennq/lsp-powershell
+;; Package-Version: 20190420.1258
```

Comment:
I don't think Quelpa need version information, can you please remove this? If possible I want to keep stable version instead of this one.
With your conversion of file ending to LF, Quelpa should work right?